packages/test-exclude/index.js

```diff
@@ -180,7 +180,7 @@ exportFunc.defaultExclude = [
     '**/*{.,-}test.js',
     '**/__tests__/**',
     '**/node_modules/**',
-    '**/babel.config.js'
+    '**/{babel,nyc}.config.js'
```

Comment:
you could also add `jest`

Reply:
Sure.  Going to also add `ava`.  Any others you think?

Reply:
Not off the top of my head

Reply:
`webpack`?

Reply:
Added `rollup` as well.